ofnetBgp.go

```diff
@@ -625,3 +592,13 @@ func (self *OfnetBgp) InspectProto() (interface{}, error) {
 
 	return OfnetBgpInspect, nil
 }
+
+func (self *OfnetBgp) SetRouterInfo(uplink *PortInfo) {
+	log.Infof("Received Set Router info for %v \n", uplink)
+	if uplink != nil && len(uplink.MbrLinks) != 1 {
+		log.Errorf("OfnetBgp currently supports only one uplink interface. Num uplinks active: %d", len(uplink.ActiveLinks))
+		return
```

Comment:
@gkvijay @abhinandanpb: FYI, this was the failure scenario we had discussed when fwd-mode change is triggered. From our discussion, we should be handling this by picking one of the member links.